app/views/settings/kubelet_compute_resources_reservations/index.html.slim

```diff
@@ -0,0 +1,143 @@
+= render 'settings/apply'
+
+.alert.alert-danger role='alert'
+  p Warning: Entering invalid values for any of the following settings will cause
+    the nodes to enter into a broken state.
+
+h2 Compute resources reservations
+
+p Every node of the Kubernetes cluster has a kubelet instance running. By default,
+  the kubelet process will try to use all available resources on each node. This
+  behaviour can lead to resource starvation for critical system services as well
+  as for Kubernetes' own components.
+
+p To prevent this, it is possible to instruct kubelet to reserve a certain
+  amount of resources for the host system and for Kubernetes core services on each node.
+  The Kubernetes scheduler takes these limits into account by when deciding
+  on which node to schedule a certain pod.
+
+= form_for :kubelet_compute_resources_reservations, url: settings_kubelet_compute_resources_reservations_path, method: :post do |f|
+  .panel.panel-default
+
+    .panel-heading
+      h3.panel-title Kubernetes core services
+    .panel-body
+      p This category include processes such as:
+      ul
+        li kubernetes API server
+        li kubernetes controller manager
+        li kubernetes scheduler
+        li kubelet
+        li kube-proxy
+        li Container runtime: Docker daemon, containerd, CRI-O or runc
+
+      .form-group
+        = f.label :cpu, "CPU"
+        .input-group
+          = f.text_field :kube_cpu, value: @kube.cpu, class: 'form-control', 'aria-describedby' => 'kube_cpu_help'
+          small.form-text.text-muted#kube_cpu_help
+            | The amount of CPU units to reserve, as a decimal number or in "millicores" (e.g. 
+            code 100m
+            |  , 
+            code 0.1
+            | ). Leave empty for no CPU reservation.
+
+      .form-group
+        = f.label :memory, "Memory"
+        .input-group
+          = f.text_field :kube_memory, value: @kube.memory, class: 'form-control', 'aria-describedby' => 'kube_memory_help'
+          small.form-text.text-muted#kube_memory_help
+            | The amount of memory to reserve, measured in bytes (e.g. 
+            code 1024
+            |  , 
+            code 1G
+            |  , 
+            code 1Gi
+            | ). Leave empty for no memory reservation.
+
+      .form-group
+        = f.label :storage, "Ephemeral storage"
+        .input-group
+          = f.text_field :kube_ephemeral_storage, value: @kube.ephemeral_storage, class: 'form-control', 'aria-describedby' => 'kube_ephemeral_storage_help'
+          small.form-text.text-muted#kube_ephemeral_storage_help
+            | The amount of ephemeral storage measured in bytes to reserve (eg: 
+            code 1024
+            |  , 
+            code 1G
+            |  , 
+            code 1Gi
+            | ). Leave empty for no ephemeral storage reservation.
+
+    .panel-heading
```

Comment:
The css panel component wasn't done to support multiple heading and body. So you could put this into its own panel and it's gonna be visually speaking more pleasant to see. Same applies for the eviction threshold part.